src/recipes/heartbeat-by-user-first-impression/config.js

```diff
@@ -33,24 +35,40 @@ let supportedLocales = [ 'de',
 ];
 
 module.exports = {
-  "nightly": {
-    restdays: 30,
-    sample: 1 / thousand,  // 1 of 1000
-    locales: supportedLocales
+  VERSION: VERSION,
+
+  surveyUrls: {
```

Comment:
no I am not going to do this, because in JS it's really painful.  Design choice.